config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -68,3 +68,38 @@ periodics:
     - name: github
       secret:
         secretName: oauth-token
+- cron: "54 * * * *"  # Every hour at 54 minutes past the hour 
+  name: ci-branchprotector
+  cluster: test-infra-trusted
+  decorate: true
+  spec: 
+    containers:
+    - name: branchprotector
+      image: gcr.io/k8s-prow/branchprotector:v20181019-08e9d55c9
```

Comment:
An alternative here would be to give it a bazel image (launcher.gcr.io/google/bazel), check out test-infra and `bazel run //prow/cmd:branchprotector` then we don't need to worry about publishing images.

Reply:
SGTM
/hold